common-service-core/src/main/java/com/ca/mfaas/utils/SecurityUtils.java

```diff
@@ -0,0 +1,173 @@
+/*
```

Comment:
I would suggest keeping this package in the Security as this is the same feature.

http://www.javapractices.com/topic/TopicAction.do?Id=205

Reply:
I prefer package-by-layer, and I wasn't sure what practices you use in project, because I found a mix of these. But, sure, if you want I'll move to security package.